tests/kernel/semaphore/semaphore/src/main.c

```diff
@@ -202,9 +202,9 @@ void test_k_sem_define(void)
 
 	/* verify the semaphore count equals to initialized value */
 	zassert_true(signal_count == SEM_INIT_VAL,
-			"semaphore initialized failed at compile time"
-			"- expected count %d, got %d",
-			SEM_INIT_VAL, signal_count);
+				"semaphore initialized failed at compile time"
```

Comment:
It looks that you added a tab to at the line. But it does not address the identation probrom. Maybe it has something to do with the setting of your editor.

Reply:
I did changes using uncrustify tool now and changed code.